purchase_request/models/stock_picking.py

```diff
@@ -13,8 +13,10 @@ def _purchase_request_picking_confirm_message_content(self, picking,
                                                           request_dict):
         if not request_dict:
             request_dict = {}
-        title = _('Receipt confirmation %s for your Request %s') % (
-            picking.name, request.name)
+        import ipdb; ipdb.set_trace()
+        title = _(
+            'Receipt confirmation ' + str(
+                picking.name) + ' for your Request ' + str(request.name))
         message = '<h3>%s</h3>' % title
```

Comment:
title = _('Recepit confirmation %s for your Request %s') % (picking.name, request.name)